iothub/device/samples/getting started/SimulatedDeviceWithCommand/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This application uses the Azure IoT Hub device SDK for .NET
+// For samples see: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples
+
+using System;
+using System.Threading.Tasks;
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    /// <summary>
+    /// This sample illustrates the very basics of a device app sending telemetry and receiving a command.
+    /// For a more comprehensive device app sample, please see
+    /// <see href="https://github.com/Azure/azure-iot-sdk-csharp/tree/previews/v2/iothub/device/samples/how%20to%20guides/DeviceReconnectionSample"/>.
```

Comment:
Be careful when linking to a specific branch on this repo. Once we merge previews/v2 into main, this link will be broken. I'd consider making this link just point to main.